pkg/lclient/fakeclient.go

```diff
@@ -39,6 +39,74 @@ var mockImageSummary = []types.ImageSummary{
 	},
 }
 
+var mockContainerJSONList = []types.ContainerJSON{
+	types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "/node",
+			Image: "node",
+			ID:    "1",
+		},
+		Mounts: []types.MountPoint{
+			{
+				Destination: OdoSourceVolumeMount,
+			},
+		},
+		Config: &container.Config{
+			Image: "node",
+			Labels: map[string]string{
+				"component": "test",
+				"alias":     "alias1",
+			},
+		},
+	},
+	types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "/go-test",
+			Image: "golang",
+			ID:    "2",
+			HostConfig: &container.HostConfig{
+				PortBindings: nat.PortMap{
+					nat.Port("8080/tcp"): []nat.PortBinding{
+						nat.PortBinding{
+							HostIP:   "127.0.0.1",
+							HostPort: "54321",
+						},
+					},
+				},
+			},
+		},
+	},
+	types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "/go-test-build",
+			Image: "golang",
+			ID:    "3",
+			HostConfig: &container.HostConfig{
+				PortBindings: nat.PortMap{
+					nat.Port("8080/tcp"): []nat.PortBinding{
+						nat.PortBinding{
+							HostIP:   "127.0.0.1",
+							HostPort: "65432",
+						},
+					},
+				},
+			},
+		},
+		Mounts: []types.MountPoint{
+			{
+				Destination: OdoSourceVolumeMount,
+			},
+		},
+		Config: &container.Config{
+			Image: "golang",
+			Labels: map[string]string{
+				"component": "test",
+				"alias":     "alias1",
```

Comment:
The url value is missing https://github.com/openshift/odo/blob/c3ef331b4b85dec9edbf4c86071e84b68ac44422/pkg/lclient/fakeclient.go#L82